### ISSUE TITLE
Default Android font is not used

### ISSUE THREAD

**Issue by siSSSou** [readrops/Readrops#197]
On my phone I set up Roboto Condensed as the default font. This font is used everywhere.
... Except in Readrops (beta 2) : content article is displayed with another font !?

![Screenshot_20240912-203913_ReadropsBeta](https://github.com/user-attachments/assets/fa0eefaa-005d-434f-aa3c-87d5e602cb2a)

**Comment by Shinokuni** [comment]
Yeah that's intended. I choosed this font to replace the default webview one.
